### PR TITLE
added some diagnostic ions and neutral losses to the common biological modifications text file

### PR DESCRIPTION
**Pull request by trishorts**

### CODE REVIEW

File: EngineLayer/Mods/CommonBiological.txt [hunk 1]
```diff
@@ -1,15 +1,31 @@
...
+ID   Phosphotyrosine
+TG   Y
+PP   Anywhere.
+NL   H0 or H3 O4 P1
+MT   Biological
+CF   H1 O3 P1
+DI   216.042
+//
+ID   Phosphothreonine
 TG   S or T or Y
```
Comment (rmillikin) [inline EngineLayer/Mods/CommonBiological.txt]: `TG   T` ?

File: EngineLayer/Mods/CommonBiological.txt [hunk 2]
```diff
@@ -1,15 +1,31 @@
...
+MT   Biological
+CF   H1 O3 P1
+//
+ID   Phosphotyrosine
+TG   Y
+PP   Anywhere.
+NL   H0 or H3 O4 P1
+MT   Biological
+CF   H1 O3 P1
+DI   216.042
```
Comment (rmillikin) [inline EngineLayer/Mods/CommonBiological.txt]: I don't think you want this here

File: EngineLayer/Mods/CommonBiological.txt (hunk 1 above)
Comment (trishorts) [reply EngineLayer/Mods/CommonBiological.txt]: Good  catch. Just T. I'll change tomorrow

File: EngineLayer/Mods/CommonBiological.txt (hunk 2 above)
Comment (acesnik) [reply EngineLayer/Mods/CommonBiological.txt]: Why not?

File: EngineLayer/Mods/CommonBiological.txt [hunk 3]
```diff
@@ -1,15 +1,31 @@
...
+//
+ID   Phosphothreonine
+TG   T
 PP   Anywhere.
 NL   H0 or H3 O4 P1
 MT   Biological
 CF   H1 O3 P1
 //
-ID   Acetyl_Internal
+ID   Acetyl_Internal_Lysine
```
Comment (acesnik) [inline EngineLayer/Mods/CommonBiological.txt]: Do we allow spaces in these names?

How about "Acetyllysine, Internal"?

**Pull request merged**